features/dynamic-bundle-discounts.md

```diff
@@ -0,0 +1,210 @@
+# Dynamic Bundle discounts
+
+- [Dynamic Bundle discounts](#dynamic-bundle-discounts)
+  - [Introduction](#introduction)
+  - [Setup](#setup)
+    - [Nosto Dynamic Bundle template](#nosto-dynamic-bundle-template)
+    - [Shopify Line Item script](#shopify-line-item-script)
+    - [Known Issues](#known-issues)
+
+## Introduction
+This documentation explains the process of setting up `Nosto - Dynamic Bundles` for automatic discount, when a bundle is added to the cart on a Shopify store. As per the existing setup, the end-to-end process has to be done manually. This release improves this by reducing manual intervention, to an extent, in setting up the store for bundle discounts.
+
+## Setup
+
+### Nosto Dynamic Bundle template
+A new Nosto JavaScript API `addBundleToCartWithDiscount` has been introduced. Functionalities of this API are as follows, 
+- collect the bundle information, from the bundle template, 
+- **validate and prepare the data for Shopify API call** (This step adds discount details along with bundle information to line item properties)
```

Comment:
Why is the beginning of this sentence in bold? Using random different formatting makes it more difficult and confusing to read.

Reply:
The final version is not going to be like this. Only the MD version has the extra characters. It conveys an important point and hence it's highlighted